projects/plugins/jetpack/extensions/blocks/nextdoor/utils.js

```diff
@@ -0,0 +1,33 @@
+export const REGEX = /(^|\/\/|www.)(nextdoor\.[^"']*)/i;
```

Comment:
I think that dot after www should be escaped:
```suggestion
export const REGEX = /(^|\/\/|www\.)(nextdoor\.[^"']*)/i;
```